src/Tests/Microsoft.NET.Build.Containers.UnitTests/DockerDaemonAvailableUtils.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.NET.Build.Containers.UnitTests;
 
 public class DockerDaemonAvailableTheoryAttribute : TheoryAttribute
 {
-    private static bool IsDaemonAvailable = new LocalDocker(Console.WriteLine).IsAvailable();
+    private static bool IsDaemonAvailable = new LocalDocker(message => Console.WriteLine($"{message.level}: {String.Format(message.messageFormat, message.formatArgs)}")).IsAvailable();
```

Comment:
minor: the `ILogger` scheme allows for different providers, Console included.

Reply:
Changed in favor of `ILogger` implementation